unitTestSupportFunctions/calculateLift.m

```diff
@@ -0,0 +1,30 @@
+%% Function documentation
```

Comment:
Follow my coding style. First comes the function signature, then the licensing information and then the function documentation